docs/docs/guides/automate/declarative-automation/automating-asset-checks.md

```diff
@@ -0,0 +1,50 @@
+---
+description: Guide on automating AssetChecks
+sidebar_position: 100
+title: Automating asset checks
```

Comment:
Instead of adding this page to the Declarative Automation docs, I recommend turning the "[Scheduling and monitoring asset checks](https://docs.dagster.io/guides/test/asset-checks#scheduling-and-monitoring-asset-checks)" section of the Asset checks page into a standalone page in the Test section and adding section headers for "Using declarative automation" and "Using schedules and sensors". Once that's in place, you can add a note or tip to the Declarative automation docs index page that links to the new page. (I would also recommend linking back to the Declarative Automation docs from the new page.)

Reply:
makes sense -- I ended up creating this as followup task as I don't quite have time to get this done before i leave on friday: https://linear.app/dagster-labs/issue/BCOR-206/dadocs-move-asset-check-automation-to-asset-checks-documentation